### PR TITLE
v0.2

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by duglin** [comment]
The href check failures are ok and will be fixed once this is merged.

**Comment by duglin** [comment]
If the vote passes, here's the list of major changes for v0.2 that I was going to include in the release notes - please let me know if you think I'm missing something:
- Added HTTP WebHook Specification (#155)
- Added AMQP 1.0 transport and AMQP type system mapping (#157)
- Added MQTT 3.1.1 and 5.0 transport binding (#158)
- Added NATS transport binding (#215)
- Added Distributed Tracing extension (#227)
- Added a Primer (#238)
- Added Sampling extension (#243)
- Defined minimum bar for new protocols/encodings (#254)
- Removed eventTypeVersion (#256)
- Moved serialization of extensions to be top-level JSON attributes (#277)
- Added Sequence extension (#291)
- Added Protobuf transport (#295)
- Defined minimum bar for new extensions (#308)
- Require all attributes to be lowercase and restrict the character set (#321)
- Simplified/shortened the attribute names (#339)

**Comment by duglin** [comment]
IBM votes yes on v0.2
IBM LGTMs this PR (just to show the pattern, I can't technically vote for my own PR :-) )

**Comment by rperelma** [comment]
Adobe votes yes on v0.2

File: roadmap.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ item; they don't imply an order for implementation._
     1. Events are sent by code written by Developer1 and received by code
        written by Developer2, where Developer1 has no knowledge of Developer2.
 
-*0.2*
+*0.2* - Completed - 2018/12/05
```
Comment (cneijenhuis) [inline roadmap.md]:
```suggestion
*0.2* - Completed - 2018/12/06
```

Super-minor, but I think the vote officially closes tomorrow? 😉

Comment (duglin) [reply roadmap.md]: clearly calendaring isn't my thing :-)  will fix, thanks

**Comment by Vlaaaaaaad** [comment]
I'm voting yes on v0.2. 

I'd really like to see https://github.com/cloudevents/spec/pull/356 merged before we cut 0.2 so we can highlight the SDK effort and evolution even more.

**Comment by duglin** [comment]
Technically the vote for v0.2 does not include that PR. However, since the PR doesn't impact any of our normative docs and is strictly additive, if the group wants to include it as part of the release I think we can do so. I'll try to remember to bring up this option on today's call.

**Comment by deissnerk** [comment]
SAP votes yes for v0.2

**Comment by clemensv** [comment]
Microsoft is yes for 0.2

**Comment by duglin** [comment]
Vote passed with 12 yes votes and 0 no votes.
We also agreed to merge https://github.com/cloudevents/spec/pull/356 before we cut the release.

The yes votes were from:
Adobe, commercetools, Google, IBM, Microsoft, Nats/Synadia, Nordstrom, Oracle, Red Hat, SAP, VMWre, Vlad Ionescu

**Comment by duglin** [comment]
Approved on the 12/6 call !!!

**Pull request merged**